src/Forms/FormField.php

```diff
@@ -490,18 +490,17 @@ public function setTitle($title)
      * Gets the contextual label than can be used for additional field description.
      * Can be shown to the right or under the field in question.
      *
-     * @return string Contextual label text.
+     * @return string Contextual label text (HTML escaped)
      */
     public function RightTitle()
     {
         return $this->rightTitle;
     }
 
     /**
-     * Sets the right title for this formfield
-     * Note: This expects escaped HTML.
+     * Sets the right title for this formfield. Note: Any HTML passed in will be escaped.
```

Comment:
Rather, document that you can pass in a DBField instance of the appropriate escape type to override the default escape.

```
@param string|DBField $rightTitle Plain text right title, or a DBField with the appropriate escape type
```